### ISSUE TITLE
SOS Alert TK102

### ISSUE THREAD

**Issue by tectonny** [vitalidze/traccar-web#824]
Dear Vitaly, you can create a highlight on the map when recido an SOS alert the TK102?

It can be a circle equal to maintenance.

I did some tests here, and appears only a very quick message.

**Comment by ghost** [comment]
very good idea friend

**Comment by vitalidze** [comment]
Yes, I can create it. For now there are other tasks with higher priority and I don't have time to work even on them.

**Comment by akmal1999** [comment]
Hi Vitalidze

does traccar support tk102, I found it hard to find out how to configure it to admin panel. Meaning How do you add this as device in main admin panel. Port number is 8169, how do you configure it. I went through the support info from the tab devices... but, does not specify how to add this device to admin panel. is there a way to find this device Id.

thanks

**Comment by vitalidze** [comment]
@akmal1999 first of all please don't post your questions in random threads. This one is not related anyhow with your question.

As per your question I believe it's obvious on how to add device to the system. You should use "Add" button in the devices panel. There will be an "Unique ID" field, where you should put some identifier of your device (IMEI in some cases).
